CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Change Log
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## 2.1.0-esm.0
```

Comment:
Can you remove it? In my project changelog changes are part of release commit

Reply:
that makes sense

i don't have any time soon